pkg/cmd/subcommand/demo.go

```diff
@@ -0,0 +1,478 @@
+package subcommand
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"github.com/OpenFunction/cli/pkg/client"
+	"github.com/OpenFunction/cli/pkg/components/inventory"
+	"os"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/OpenFunction/cli/pkg/cmd/util"
+	"github.com/OpenFunction/cli/pkg/components/common"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"golang.org/x/sync/errgroup"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	k8s "k8s.io/client-go/kubernetes"
+)
+
+// Demo is the commandline for 'init' sub command
+type Demo struct {
+	genericclioptions.IOStreams
+
+	Verbose             bool
+	RegionCN            bool
+	OpenFunctionVersion string
+	DryRun              bool
+	AutoPrune           bool
+	Timeout             time.Duration
+}
+
+const DemoYamlFile = "https://raw.githubusercontent.com/OpenFunction/OpenFunction/main/config/samples/function-sample-serving-only.yaml"
+
+func init() {
+	w = os.Stdout
+}
+
+// NewDemo returns an initialized Init instance
+func NewDemo(ioStreams genericclioptions.IOStreams) *Demo {
+	return &Demo{
+		IOStreams: ioStreams,
+	}
+}
+
+func NewCmdDemo(restClient util.Getter, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	var cl *k8s.Clientset
+	i := NewDemo(ioStreams)
+
+	cmd := &cobra.Command{
+		Use:                   "demo <args>...",
+		DisableFlagsInUseLine: true,
+		Short:                 "Create OpenFunction demo.",
+		Long: `
+         You can use ofn demo to run the OpenFunction demo.
+         Available options are: --auto-prune, --region-cn, --verbose
+`,
+		Example: "ofn demo",
+		Run: func(cmd *cobra.Command, args []string) {
+			util.CheckErr(i.RunKind(cl, cmd))
+		},
+	}
+
+	cmd.Flags().BoolVar(&i.Verbose, "verbose", false, "Show verbose information.")
+	cmd.Flags().BoolVar(&i.RegionCN, "region-cn", false, "For users who have limited access to gcr.io or github.com.")
+	cmd.Flags().BoolVar(&i.AutoPrune, "auto-prune", true, "Used to specify the version of OpenFunction to be uninstalled.")
```

Comment:
"Used to specify the version of OpenFunction to be uninstalled." is not correct for auto-prune